pkg/scorecard.go

```diff
@@ -105,9 +105,25 @@ func RunScorecard(ctx context.Context,
 	defer repoClient.Close()
 
 	commitSHA, err := getRepoCommitHash(repoClient)
-	if err != nil || commitSHA == "" {
+
+	if errors.Is(err, sce.ErrorEmptyRepository) {
+		versionInfo := version.GetVersionInfo()
+		ret := ScorecardResult{
+			Repo: RepoInfo{
+				Name:      repo.URI(),
+				CommitSHA: commitSHA,
+			},
+			Scorecard: ScorecardInfo{
+				Version:   versionInfo.GitVersion,
+				CommitSHA: versionInfo.GitCommit,
+			},
+			Date: time.Now(),
```

Comment:
we have some code below that does the same thing. What do you think about moving it higher so we only define it in one spot?

Reply:
good idea. moved this higher too